app copy/views/posts/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Members Posts</h1>
+
+<div class= "row">
+ <div class="col-md-6 col-md-offset-3">
+    <% if logged_in? %>
+      <%= link_to "New Post", new_post_path, class: 'post-new btn btn-primary' %>
+    <% end %>
+
+    
+    <%# <img src="img_avatar3.png" alt="John Doe" class="mr-3 mt-3 rounded-circle" style="width:60px;"> %>
```

Comment:
Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂